pagopa/pn-frontend
# fix: CTA inline in domicile banner of PF also on mobile view

## Short description
CTA inline in domicile banner of PF also on mobile view

## List of changes proposed in this pull request
- Modified Domicile Banner component

## How to test
Start PF and go to Notification page, resize page on mobile view and CTA on domicile banner should be inline
